README.md

```diff
@@ -44,20 +44,22 @@ Configuration can be provided via:
+| TESTRAIL_REFERENCE            | reference          | String that will be added to the `refs` field in TestRail. This can enable integration with other tools like https://github.com/DamianOsipiuk/jest-reporter-testrail/. Configurable with variables <ul><li>`%BRANCH%` - see config option `branchEnv`</li><li>`%BUILD%` - see config option `buildNoEnv`</li></ul>     |                             |          |
+| TESTRAIL_BRANCH_ENV           | branchEnv          | Which ENV variable is used to store branch name on which tests are run.                                                                                                                                                                                                                                                |          `BRANCH`           |          |
+| TESTRAIL_BUILD_NO_ENV         | buildNoEnv         | Which ENV variable is used to store build number of tests run.                                                                                                                                                                                                                                                         |       `BUILD_NUMBER`        |          |
+| TESTRAIL_DATE_FORMAT          | dateFormat         | What date format should be used for `%DATE%` placeholder. https://momentjs.com/ formats supported.                                                                                                                                                                                                                     |    `YYYY-MM-DD HH:mm:ss`    |          |
+| TESTRAIL_RUN_CLOSE_AFTER_DAYS | runCloseAfterDays  | After how many days should reporter close old Runs in testrail.                                                                                                                                                                                                                                                        |                             |          |
+| TESTRAIL_UPLOAD_SCREENSHOTS   | uploadScreenshots  | Should upload screenshots to testrail. Requires test result edit enabled in testrail.                                                                                                                                                                                                                                  |           `false`           |          |
+| TESTRAIL_RUN_ID               | runId              | Run id which test cases are stored. Ex `R123`                                                                                                                                                                                                                                                                          |                             |          |
```

Comment:
Could you move this flag under **project_id**? I feel it will be more concise this way.